test/integration/script-loader/test/index.test.js

```diff
@@ -187,37 +185,39 @@ const runTests = (isDev = false) => {
     }
   })
 
-  if (!isDev) {
-    it('priority beforeInteractive with inline script', async () => {
-      const html = await renderViaHTTP(appPort, '/page5')
-      const $ = cheerio.load(html)
+  it('priority beforeInteractive with inline script', async () => {
+    const html = await renderViaHTTP(appPort, '/page5')
+    const $ = cheerio.load(html)
 
-      const script = $('#inline-before')
-      expect(script.length).toBe(1)
+    const script = $('#inline-before')
+    expect(script.length).toBe(1)
 
+    // css bundle is only generated in production, so only perform inline script position check in production
+    if (!isDev) {
       // Script is inserted before CSS
       expect(
         $(`#inline-before ~ link[href^="/_next/static/css"]`).length
       ).toBeGreaterThan(0)
-    })
+    }
```

Comment:
Note that there is no `/_next/static/css` bundle generated in dev mode (HMR CSS is used instead), so this `expect` is skipped in dev.